tunnel/IpTunnel.c

```diff
@@ -127,14 +127,14 @@ static struct IpTunnel_Connection* connectionByPubKey(uint8_t pubKey[32],
 int IpTunnel_allowConnection(uint8_t publicKeyOfAuthorizedNode[32],
                              struct Sockaddr* ip6Addr,
                              uint8_t ip6Prefix,
+                             uint8_t ip6NetworkSize,
                              struct Sockaddr* ip4Addr,
                              uint8_t ip4Prefix,
+                             uint8_t ip4NetworkSize,
                              struct IpTunnel* tunnel)
 {
     struct IpTunnel_pvt* context = Identity_check((struct IpTunnel_pvt*)tunnel);
 
-    Log_debug(context->logger, "IPv4 Prefix to allow: %d", ip4Prefix);
```

Comment:
Because it was random debug info (only IPv4 prefix). If it is useful I will readd it with more info.